README.md

```diff
@@ -72,6 +72,48 @@ function (request, reply) {
 }
 ```
 
+# Database Structure
+
+### **Tags**
+This table will list the details of _**all** tags_.
+
+Each tag will detail:
+
+| field  | data-type    |
+| --------------------- |
+| id     | serial       |
+| name   | varchar(50)  |
+| active | boolean      |
+
+### **Tag-Categories**
+This table will list the details of the **groups** or **types** of tags.
+
+Each group will detail:
+
+| field  | data-type    |
+| --------------------- |
+| id     | serial       |
+| name   | varchar(50)  |
+| active | boolean      |
+
+### **Categories-Tags**
+This table will detail the relationships between a tag and a group; i.e. it will list which _tag_ belongs in which _group_.
+
+Each group will detail:
+
+| field            | data-type    |
+| ------------------------------- |
+| category- id     | varchar(50)  |
```

Comment:
category-id and tag-id reference the keys id of the tags and categories so the type should be Integer (serial use this type I think but I will check again)